qiskit/scheduler/schedule_circuit.py

```diff
@@ -32,18 +32,22 @@ def schedule_circuit(circuit: QuantumCircuit,
     scheduled to occur as late as possible.
 
     Supported methods:
-        'as_soon_as_possible': Schedule pulses greedily, as early as possible on a qubit resource
-            alias: 'asap'
-        'as_late_as_possible': Schedule pulses late-- keep qubits in the ground state when possible
-            alias: 'alap'
+
+        * ``as_soon_as_possible``: Schedule pulses greedily, as early as possible on a qubit resource.
```

Comment:
```suggestion
        * ``'as_soon_as_possible'``: Schedule pulses greedily, as early as possible on a qubit resource.
```
since it is a string